uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/PromptParameter.java

```diff
@@ -29,5 +29,6 @@
     SELECT_ACCOUNT,
     LOGIN,
     CONSENT,
+    CREATE,
```

Comment:
Please add at the end of the enum list

Reply:
I mean all new items should generally go at the bottom of the list to avoid disturbing the ordinal value. (We don't actually use the ordinal values of these enums but still a good practice to follow)

Reply:
Oh, I actually did not know that, thanks!